PULL_REQUESTS.md

```diff
@@ -59,6 +59,9 @@ If there are documentation changes, please include a brief description of what t
 changes may be in [docs/root](docs/root) and/or inline with the API protos. Please write in
 N/A if there were no documentation changes.
 
+Any structural changes to the dataplane should also update the [Life of a
```

Comment:
super nitty but either
Any structural changes -> Any PRs with structural changes
or
should also update the -> should land with updates to the